Makefile

```diff
@@ -189,6 +189,7 @@ k0s.exe: TARGET_OS = windows
 k0s.exe: BUILD_GO_CGO_ENABLED = 0
 
 k0s.exe k0s: $(GO_SRCS) $(codegen_targets) go.sum
+	-[ ! -e '$@' ] || rm -- '$@'
```

Comment:
Why not do a simple: `rm -f -- '$@'`?

Reply:
Because that will give an error when the file is not there. Such things usually catch my eyes in build outputs and then I need to spend some brain cycles to figure out that this is actually fine. Avoiding error messages is hence an investment in the future :upside_down_face:

Reply:
Oh. Actually it will not give an error if the file doesn't exist. Will change that ASAP :see_no_evil: